src/main/resources/templates/users/admin/health.ftlh

```diff
@@ -108,13 +108,13 @@
                     <h4 class="panel-title"><@spring.message "admin.health.missingFile" /></h4>
                 </div>
                 <div class="panel-body list-group list-group-health">
-                <#--@missingFileProjects.map { case (version, project) =>
-                <div class="list-group-item">
-                    <a href="${routes.getRouteUrl("versions.show", project.ownerName, project.slug, version.name)}">
-                        <strong>${project.namespace}/${version.name}</strong>
-                </a>
-                </div>
-                }--> <#--TODO missinFileProjects -->
+                    <#list missingFileProjects as missingFileProject>
+                        <div class="list-group-item">
+                            <a href="${routes.getRouteUrl("versions.show", missingFileProject.getOwner(), missingFileProject.getName(), missingFileProject.getVersion().getVersionString())}">
```

Comment:
same, here. can't you use the freemarker format of `missingFileProject.owner`, `missingFileProject.name`, etc.

Reply:
Yea, I change it here. Forgot to change it back. new commit contains the change